models/dataSource.js

```diff
@@ -0,0 +1,20 @@
+const { DataSource } = require("typeorm");
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_NAME,
+});
+
+appDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  })
+   .catch((err)=> { 
+      console.error("Failed to connect to Database")
```

Comment:
- 해당 코드를 TypeOrm 공식문서에서 제공하는 포맷과 동일하게 잘 작성해주셨습니다.
- 다만 아쉬운 점은, tab을 이용한 들여쓰기와 스페이스바를 이용한 공백처리가 모두 포함되어 있는 것 같습니다.
동일한 코드 블록을 가리키기 위해 동일한 들여쓰기를 지키며 코드를 작성하는 것도 가독성을 유지하는데 중요한 요소이기 때문에 이 부분도 반드시 확인해주세요~!
```suggestion
appDataSource.initialize()
  .then(() => {
    console.log("Data Source has been initialized!")
  })
  .catch((err)=> {
    console.error("Failed to connect to Database")
```